apps/prairielearn/src/lib/editors.js

```diff
@@ -1386,30 +1386,20 @@ export class FileModifyEditor extends Editor {
 export class CourseInfoEditor extends Editor {
   constructor(params) {
     super(params);
-    this.description = `Create infoCourse.json`;
+    this.description = `Write infoCourse.json`;
```

Comment:
I think we actually want to revert this change and the `commitMessage` change below, since we're truly only using this to create new files. Maybe name this `CourseInfoCreateEditor` to make it clear from the name?

Reply:
Reverted changes here and changed name to `CourseInfoCreateEditor` in [1f9d924](https://github.com/PrairieLearn/PrairieLearn/pull/9581/commits/1f9d9242b1ab757397ce2a399f68924f58891b8e).